stellar_core_prometheus_exporter/exporter.py

```diff
@@ -411,4 +413,11 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    try:
+        main()
+    except KeyboardInterrupt:
+        print('Interrupted')
+        try:
+            sys.exit(0)
+        except SystemExit:
+            os._exit(0)
```

Comment:
JOOI what's the reason for both sys.exit and os._exit?

I'm also wondering if we even need to exit explicitly, I _think_ it may be enough to catch the exception and just `pass` but it's worth testing
```suggestion
    except KeyboardInterrupt:
        print('Interrupted')
```